src/main/java/com/example/payroll/Models/Employees.java

```diff
@@ -0,0 +1,31 @@
+package com.example.payroll.Models;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class Employees {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Long employee_id;
+
+    private String first_name;
+    private String last_name;
+    private String department;
+    private String position;
+    private String email;
+
+    private Boolean employee_status;
+    private Boolean pf_status;
```

Comment:
We should rename to more boolean-like name
For example `is_pension_payer`

Reply:
done